_posts/2018-06-29-14-04-netcore-7-0.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title : "14.4 OData Web API 7.0 now available"
```

Comment:
I think we should take out "now available" due to the style of the docs that we have (in line with Saurabh's comment regarding taking out "today"). If we had a blog, I think the phrase would be more appropriate.

Reply:
#resolved.